pkgs/top-level/all-packages.nix

```diff
@@ -17639,7 +17639,9 @@ in
 
   dodgy = with python3Packages; toPythonApplication dodgy;
 
-  dovecot = callPackage ../servers/mail/dovecot { };
+  dovecot = callPackage ../servers/mail/dovecot {
+    lua = lua5_4;
```

Comment:
This is not necessary, but it builds with it and our lua defaults to 5.2 for some reason, which has been EOL since 2015.

Reply:
Maybe we should update the default lua version.